org.eclipse.xtext.testing/src/org/eclipse/xtext/testing/AbstractLanguageServerTest.xtend

```diff
@@ -407,6 +409,10 @@ abstract class AbstractLanguageServerTest implements Endpoint {
 	protected dispatch def String toExpectation(CodeLens it) {
 		return command.title + " " +range.toExpectation
 	}
+	
+	protected dispatch def String toExpectation(FoldingRange it) {
+		return toString
```

Comment:
Question: For CodeLens above we do invoke toExpectation on the range. Should we do something similar for the positions in the range?

Reply:
Right, that simplified folding range testing a lot :D